src/main/frontend/components/NotFoundPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+class NotFoundPage extends React.Component{
+    render(){
+        return <div className="error-page">
+            <img src={"https://miro.medium.com/max/1068/1*wUOrpv-selJOytCkslSIhg.png"} className="pizza-image" />
+            <h2>Page Not Found!</h2>
+          </div>;
+    }
+}
```

Comment:
The indentation looks different here.  Double check the spacing.  It may be a tabs vs spaces issue

Reply:
Fixed the spacing